Makefile

```diff
@@ -362,8 +362,8 @@ setup-local-debugging: ## 🛠️ Setup local debugging
 	&& . ${MAKEFILE_DIR}/devops/scripts/load_env.sh ${MAKEFILE_DIR}/core/private.env \
 	&& . ${MAKEFILE_DIR}/devops/scripts/setup_local_debugging.sh
 
-auth: ## 🔐 Create the necessary Azure Active Directory assets
-	$(call target_title,"Setting up Azure Active Directory") \
+auth: ## 🔐 Create the necessary Microsoft Entra Workforce ID assets
```

Comment:
I think need to do a find and replace for "Microsoft Entra Workforce ID" -> Microsoft Entra ID.